jme3-welcome-screen/src/com/jme3/gde/welcome/WelcomeScreenTopComponent.java

```diff
@@ -57,7 +56,7 @@ public WelcomeScreenTopComponent() {
 
         jScrollPane2.setOpaque(false);
         jScrollPane2.getViewport().setOpaque(false);
-        jEditorPane1.putClientProperty(SwingUtilities2.AA_TEXT_PROPERTY_KEY, SwingUtilities2.AATextInfo.getAATextInfo(true));
```

Comment:
Looks good, but usually having separate PRs is wanted, it has a few pros.

Reply:
I will create another PR for this.